config/mpSkyEphemerisQuery.py

```diff
@@ -0,0 +1,2 @@
+config.observerCode = "W84"  # IAU code for DECam
+config.queryBufferRadiusDegrees = 0.2
```

Comment:
Is there any need to override `queryBufferRadiusDegrees`, if it's just set by proper motions?